src/react-styleguide/src/components/Checkboxes/Checkboxes.js

```diff
@@ -0,0 +1,139 @@
+import React, { Component } from 'react'
+
+export const commonLabelClasses = 'inline-flex v-top items-center lh-title f4 overflow-hidden pointer'
+export const commonIconClasses = 'mr3 ba b--white hover-b--green'
+export const inputClasses = 'o-0 absolute left--1'
+export const disabledClasses = 'disabled'
+
+
+const checkboxClasses = 'pointer eh-checkbox'
+const boxIconClasses = `${commonIconClasses} br2 eh-checkbox-icon`
+
+class Checkbox extends Component {
+  static propTypes = {
+    name: React.PropTypes.string,
+    label: React.PropTypes.string,
+    checked: React.PropTypes.bool,
+    disabled: React.PropTypes.bool,
+    onChange: React.PropTypes.func, // Called when user changes checked state
+    onClick: React.PropTypes.func, // Called when checkbox is clicked
+  }
+
+  constructor(props) {
+    super(props)
+  
+    const checked = 'checked' in props ? props.checked : false
+
+    this.state = { checked: checked }
```

Comment:
nbd but can use object shorthand here of just `{checked}` since key and value are the same